.github/workflows/check-bioc.yml

```diff
@@ -0,0 +1,291 @@
+## Read more about GitHub actions the features of this GitHub Actions workflow
+## at https://lcolladotor.github.io/biocthis/articles/biocthis.html#use_bioc_github_action
+##
+## For more details, check the biocthis developer notes vignette at
+## https://lcolladotor.github.io/biocthis/articles/biocthis_dev_notes.html
+##
+## You can add this workflow to other packages using:
+## > biocthis::use_bioc_github_action()
+##
+## Using GitHub Actions exposes you to many details about how R packages are
+## compiled and installed in several operating system.s
+### If you need help, please follow the steps listed at
+## https://github.com/r-lib/actions#where-to-find-help
+##
+## If you found an issue specific to biocthis's GHA workflow, please report it
+## with the information that will make it easier for others to help you.
+## Thank you!
+
+## Acronyms:
+## * GHA: GitHub Action
+## * OS: operating system
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
```

Comment:
You may want to add `- main` here also, in case we ever switch the `master` branch to `main`. It's not necessary to do it now (and it could be done if the branch name changes), but it won't hurt anything and it would be easy to forget in the future.

Reply:
Good point. Renaming `master` to `main` is on my to-do list, but I know Matt is in the middle of working on multiple feature branches and I don't want to create additional headaches for his local environment right now.